soes/esc_coe.c

```diff
@@ -299,7 +299,7 @@ static void SDO_upload (void)
    uint16_t index;
    uint8_t subindex;
    int32_t nidx;
-   int32_t nsub;
+   int16_t nsub;
```

Comment:
Why is this changed back to int16_t?

Reply:
I think it was unnecessarily promoted to int32_t in last commit, in most cases nsub is int16_t trough out the file and can be it in SDO_upload as well,. So, the idea was to align the usage in this context and I changed it back.
No warning.